### PR TITLE
[ENH] creating 'I'm loading' animations for network based uploads- and rank-view activities

### PR DESCRIPTION
**Pull request by rksh**
explain that these two blank screens are trying to do something.

### CODE REVIEW

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ui/ProgressThrobberFragment.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import net.wigle.wigleandroid.R;
+import net.wigle.wigleandroid.util.Logging;
+
+/**
+ * Loading "throbber" image parent class for network-based fragments.
+ */
+public abstract class ProgressThrobberFragment extends Fragment {
+    protected ImageView loadingImage;
+    protected boolean animating = false;
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ui/ProgressThrobberFragment.java:18]: Does this need to be Atomic? 🤯

**Review by bobzilladev — APPROVED** [review]

**Comment by bobzilladev** [comment]
A throbbing good time!

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ui/ProgressThrobberFragment.java (hunk 1 above)
Comment (rksh) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ui/ProgressThrobberFragment.java:18]: I *think* this is only manipulated with the GUI thread (but check my work)
onCreateView
RequestCompletedListener.onTaskCompleted (see uses in WiGLEApiManager - should always post to main GUI thread)

Comment (bobzilladev) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ui/ProgressThrobberFragment.java:18]: yay for gooey, ok dokey

**Pull request merged**